go.mod

```diff
@@ -48,4 +56,8 @@ replace github.com/open-telemetry/opentelemetry-collector-contrib/receiver/signa
 
 replace github.com/open-telemetry/opentelemetry-collector-contrib/receiver/zipkinscribereceiver => ./receiver/zipkinscribereceiver
 
+replace github.com/open-telemetry/opentelemetry-collector-contrib/processor/k8sprocessor => ./processor/k8sprocessor/
+
 replace k8s.io/client-go => k8s.io/client-go v0.0.0-20190620085101-78d2af792bab
+
+replace github.com/open-telemetry/opentelemetry-collector => github.com/owais/opentelemetry-collector v0.1.1-0.20200108145118-7d7e1c8f7254
```

Comment:
Why this replacement?

Reply:
Needed for CI until the other PR gets merged upstream

Reply:
Please merge this only after upstream is merged and this replacement is removed.